src/ert/gui/simulation/run_dialog.py

```diff
@@ -397,11 +397,13 @@ def _on_simulation_done(self, failed: bool, msg: str) -> None:
         self.kill_button.setHidden(True)
         self.restart_button.setVisible(self._run_model.has_failed_realizations())
         self.restart_button.setEnabled(self._run_model.support_restart)
-        self.update_total_progress(1.0, msg)
         self._notifier.set_is_simulation_running(False)
         if failed:
+            self.update_total_progress(1.0, "Failed")
             self.fail_msg_box = ErtMessageBox("ERT experiment failed!", msg, self)
             self.fail_msg_box.exec_()
+        else:
+            self.update_total_progress(1.0, msg)
```

Comment:
Connected to previous comment, just do:
```suggestion
            self.update_total_progress(1.0, "Experiment completed.")
```

Reply:
Sure. If this is the message we expect to see when done, lets do that instead.